.changeset/curly-carrots-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/explorer": patch
+---
+
+The SQL editor now displays an indicator for unsaved changes.
```

Comment:
what does "unsaved" mean? makes me think that can we save queries, but I don't think that's true? maybe rephrase a bit?

```suggestion
The SQL editor now shows an indicator when the query has changed but has not been run.
```

Reply:
Yeah this sounds better, thanks!